### PR TITLE
fix(core,platform): add keyboard support for Inline Help icon triggers

### PR DESCRIPTION
**Pull request by OleksiyDrozdov**
#### Please provide a link to the associated issue.
Closes [#3441](https://app.zenhub.com/workspaces/fundamental-library-60761c0894e25c0011e828b4/issues/sap/fundamental-ngx/3441)

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [n/a] update `README.md`
- [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [n/a] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
the changes are in both repos, core and platform. Please correct the name of the PR

**Review by InnaAtanasova — APPROVED** [review]

**Review by platon-rov — APPROVED** [review]

**Review by kavya-b — APPROVED** [review]
LGTM, but maybe we also need it to work on https://deploy-preview-5862--fundamental-ngx.netlify.app/#/core/inlineHelp this page?

File: libs/core/src/lib/form/form-label/form-label.component.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export class FormLabelComponent implements OnChanges {
     /** The trigger events that will open/close the inline help.
      *  Accepts any [HTML DOM Events](https://www.w3schools.com/jsref/dom_obj_event.asp). */
     @Input()
-    inlineHelpTrigger: string[] = ['mouseleave', 'mouseenter'];
+    inlineHelpTrigger: string[] = ['mouseleave', 'mouseenter', 'focusin', 'focusout'];
```
Comment (aberikashvili) [inline libs/core/src/lib/form/form-label/form-label.component.ts:58]: I saw the same code in different place as well, if we have to add it to more and more places, may be it's better to extract it somewhere as a general setting and use everywhere? 

https://github.com/SAP/fundamental-ngx/pull/5863/files#diff-476fa704b9e826c820a27653c17d7ae879415a32f504c7e8e603ee063722400aR21

Comment (OleksiyDrozdov) [reply libs/core/src/lib/form/form-label/form-label.component.ts:58]: Screenshot 2021-07-08 195116
@aberikashvili 
I can add 'focusin', 'focusout' into inline-help.directive. Please advise if it is good idea.

**Review by aberikashvili — APPROVED** [review]
LGTM

**Pull request merged**